openpype/hosts/maya/api/lib.py

```diff
@@ -319,15 +319,15 @@ def collect_animation_data(fps=False):
     handle_start = cmds.playbackOptions(query=True, animationStartTime=True)
     handle_end = cmds.playbackOptions(query=True, animationEndTime=True)
 
-    handle_start = frame_start - handle_start
-    handle_end = handle_end - frame_end
+    frame_start_handle = frame_start - handle_start
+    frame_end_handle = handle_end - frame_end
```

Comment:
This is actually incorrect naming. In this case it's actually the `handle_start` variables above that are `frame_start_handle`. So the names here are reversed.

This is especially noticeable in the assignment below because `handleStart` should not equal `frame_start_handle` but should equal (by name) with `handle_start`.

So I'd revert both these variable name changes here and below and then change the variables above queried from maya `animationStartTime` and `animationEndTime` to be the `frame_start_handle`, etc.

Reply:
By just looking at this, shouldn't it be `handle_end + frame_end`, + and not - ?